### ISSUE TITLE
Develop guide for running Rstudio on Klone for handbook 

### ISSUE THREAD

**Issue by sr320** [RobertsLab/resources#1977]
Possible resource: https://github.com/merlab-uw/Klone/tree/main

**Comment by sr320** [comment]
also https://hyak.uw.edu/docs/tools/r#rstudio-container-and-graphical-user-interface

**Comment by kubu4** [comment]
Previous Mox guide does _not_ work.

I'll create functional documentation for Klone, based off of [the Research Computing website](https://hyak.uw.edu/docs/tools/r). Testing of a [modified version of that](https://github.com/RobertsLab/resources/issues/1978#issuecomment-2403691371) was successful (for me), so it seems good-to-go.

**Comment by kubu4** [comment]
Alrighty, I have a basic Klone-specific documentation up: https://robertslab.github.io/resources/klone_RStudio-Server/

However, I've also been working on creating a singular container which contains all programs we might want to use, _as well as_ RStudio Server. Meaning, when launching RStudio Server on Klone, it would serve as a "one stop computing" resource for running all types of analyses.

I'll update the documentation to reflect that as I get things set up and tested.

**Comment by kubu4** [comment]
I've created "one container to rule them all", which can be used via RStudio or via traditional SLURM job submission. I've updated documentation to try to reflect that. Obviously, the container contents will be updated as we decide to install additional programs/packages. 

I've set up the container builds in an attempt to be reproducible. As such, each time the container is built, it's name is slightly different (incorporates the commit hash into filename), so can't write documentation that's just copy/paste.
